src/Microsoft.ML.Transforms/Text/TextNormalizing.cs

```diff
@@ -212,6 +213,31 @@ public Mapper(TextNormalizingTransformer parent, DataViewSchema inputSchema)
                 }
             }
 
+            public bool CanSaveOnnx(OnnxContext ctx) => true;
+            public void SaveAsOnnx(OnnxContext ctx)
+            {
+                Host.CheckValue(ctx, nameof(ctx));
+                for (int iinfo = 0; iinfo < _types.Length; ++iinfo)
+                {
+                    string inputColumnName = _parent.ColumnPairs[iinfo].inputColumnName;
+                    if (!ctx.ContainsColumn(inputColumnName))
+                        continue;
+
+                    string outputColumnName = _parent.ColumnPairs[iinfo].outputColumnName;
+                    string srcVariableName = ctx.GetVariableName(inputColumnName);
+                    string dstVariableName = ctx.AddIntermediateVariable(_types[iinfo], outputColumnName, true);
+                    SaveAsOnnxCore(ctx, srcVariableName, dstVariableName);
+                }
+            }
+
+            private void SaveAsOnnxCore(OnnxContext ctx, string srcVariableName, string dstVariableName)
+            {
```

Comment:
The normalizer has two other options for keepDiacritics and keepPunctuations. What should happen if either of these options are specified in the pipeline? As things stand, the output of ML.NET and ORT will be different.